.github/workflows/test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Export the env variables file
         run: |
           cp .env.example .env
-          export $(egrep -v '^#' .env | xargs)
+          set -a; source .env; set +a
```

Comment:
Quite a trickery to load an `.env` file. There are a few gotchas in case of whitespaces in the variable names. They should be quoted to make this work.

Reply:
some are suggesting to rather use `export $(grep -v '^#' .env | xargs -d '\n')`
https://stackoverflow.com/a/20909045/1548052

Reply:
Didn't work with some of the variables, already tried that.